libbeat/common/cfgwarn/cfgwarn.go

```diff
@@ -0,0 +1,24 @@
+package cfgwarn
+
+import (
+	"fmt"
+
+	"github.com/elastic/beats/libbeat/logp"
+)
+
+// Beta logs the usage of an beta feature.
+func Beta(format string, v ...interface{}) {
```

Comment:
I introduced package cfgwarn to combine all configuration related warning messages in. Plus I find logp kind of messy right now.